### DISCUSSION TITLE
How does LazyVim load from `lua/plugins/*.lua`?

### DISCUSSION THREAD

**Discussion by technicalpickles** [LazyVim/LazyVim#110] [Q&A] [answered]
I have been looking, and it doesn't look like a feature of [lazy.nvim](https://github.com/folke/lazy.nvim), and I haven't been able to find anything in LazyVim that looks to be doing?

I'd like to ~steal~ borrow for my own configs 😁

**Comment by luxus** [comment]
i'm sure it is a lazy.nvim feature.
the next version 3.0 of astronvim uses it
https://github.com/AstroNvim/AstroNvim/tree/v3

**Reply by technicalpickles** [reply]
Thanks, that helped me track it down ✨

**Comment by technicalpickles** [comment, marked as answer]
Was able to track it down! It's `import` for `require("lazy").setup`, ie:

```lua
require("lazy").setup({
  import = "plugins" -- name of directory to import
})
```

This is documented in [↩️ Importing Specs, config & opts](https://github.com/folke/lazy.nvim#%EF%B8%8F-importing-specs-config--opts). Sorry for missing it, there are a lot of docs 😁
